beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -772,7 +772,7 @@ func (s *Server) GetCommittees(w http.ResponseWriter, r *http.Request) {
 
 	isOptimistic, err := helpers.IsOptimistic(ctx, []byte(stateId), s.OptimisticModeFetcher, s.Stater, s.ChainInfoFetcher, s.BeaconDB)
 	if err != nil {
-		http2.HandleError(w, "Could not check if slot's block is optimistic: "+err.Error(), http.StatusInternalServerError)
+		http2.HandleError(w, "Could not check optimistic status: "+err.Error(), http.StatusInternalServerError)
```

Comment:
this is formated differently from the above `http2.HandleError(w, errors.Wrap(err, "Could not check optimistic status").Error(), http.StatusInternalServerError)`

Reply:
fixed